pkg/lib/queueinformer/queueinformer_operator.go

```diff
@@ -197,6 +199,17 @@ func (o *operator) Run(ctx context.Context) {
 	})
 }
 
+func (o *operator) ResyncInformers() error {
```

Comment:
No client using LIST + WATCH should ever have to do this, FWIW.

Reply:
In case you're curious: the issue is we have a situation in which the namespaces gets nuked and immediately recreated with the subscription, and we're ending up with unsat...

Reply:
Are you using UID checks to detect deletion + recreation? Do you care that it deleted & recreated, or no?

Reply:
in the end the RC was: the typed queues were of _any_ type and we were enqueuing two different types of objects (string and ResourceEvent). Two semantically equal events come in and get sync'ed concurrently because we break dedupe =P